zoom_map.py

```diff
@@ -79,26 +88,57 @@ def add_level(level):
                 [("<Button-4>", partial(self._map_zoom, -1)),
                  ("<Button-5>", partial(self._map_zoom,  1)),
                  ("<Button-1>", self._on_click),
-                 ("<B1-Motion>", self._on_drag)]]
+                 ("<B1-Motion>", self._on_drag),
+                 ("<ButtonRelease-1>", self._on_unclick)]]
 
     def _set_image(self):
-        self._tk_image = self.create_image(0, 0, anchor=tk.NW,
-                                           image=self.image)
+        if self._tk_image is not None:
+            # Clean up the old image.
+            self.delete(self._tk_image)
```

Comment:
Good enough for me.  If this is something that the API docs specify, I'm ok to leave it in.  Just surprised me.